### ISSUE TITLE
Not restoring book notes saved in json file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#685]
```
What steps will reproduce the problem?
1.create a few notes in any pdf- book
2.save settings in json file. I see my notes in that json file
3.clear last boks list, cache and notes : we see empty list
4. Restore from the saved settings : I see a last books list restored but no 
saved notes when reopen a choosen book

What is the expected output?
I expect to see my notes restored from the json file

What do you see instead?
No notes at all

What version of the product are you using?
2.1.0


On what operating system?


What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `ar...@yandex.ru` on 3 Jan 2014 at 1:57

**Comment by GoogleCodeExporter** [comment]
```
Sorry, i didn't see last questions.
OS android 4.2 , devise Prestigio multipad quantum 9.7
```

Original comment by `ar...@yandex.ru` on 3 Jan 2014 at 2:00

**Comment by GoogleCodeExporter** [comment]
```
By the way! Bookmarks have been restored! Only notes (the most necessary 
information!) have been lost 
```

Original comment by `ar...@yandex.ru` on 3 Jan 2014 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
Sorry, i didn't see last questions.
OS android 4.2 , devise Prestigio multipad quantum 9.7
```

Original comment by `ar...@yandex.ru` on 3 Jan 2014 at 3:58

**Comment by GoogleCodeExporter** [comment]
```
Ok. We will check it. But at least you did not lost them - they are present in 
json. We just need to fix importing, right?
```

Original comment by `Andrei.K...@gmail.com` on 3 Jan 2014 at 9:26

**Comment by GoogleCodeExporter** [comment]
```
Yes, it seems import doesn't work.
```

Original comment by `ar...@yandex.ru` on 8 Jan 2014 at 10:11

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1.1.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 9 Jan 2014 at 8:01
- Changed state: **Fixed**
